webapp/src/Service/ScoreboardService.php

```diff
@@ -139,15 +134,11 @@ public function getTeamScoreboard(Contest $contest, int $teamId, bool $jury = fa
         $problems   = $this->getProblems($contest);
         $rankCache  = $this->getRankcache($contest, $team);
         $scoreCache = $this->getScorecache($contest, $team);
-        if ($jury || !$freezeData->showFrozen()) {
-            $teamRank = $this->calculateTeamRank($contest, $team, $rankCache, $freezeData, $jury);
-        } else {
-            $teamRank = 0;
-        }
+        $teamRank   = $this->calculateTeamRank($contest, $team, $rankCache, $freezeData, true);
```

Comment:
Are we now showing the team its rank during the freeze?

Reply:
No. We show a ‘?’ But that is handled elsewhere

Reply:
Hmmm... I think that it's better to still handle it here in backend, so that we have all the logic in a single place. Frontend can do the rendering logic of showing the actual `?`, but I think it shouldn't be responsible for what we're allowed to show.

Reply:
So you want to move the code here? That code that was here was never used.
However if we change `showFtsInFreeze` to something like `showFreezeInfo` we can basically reuse that variable to output a 0?